source/core/NumericString.cpp

```diff
@@ -1109,13 +1179,20 @@ void RefactorLabviewNumeric(const FormatOptions* formatOptions, char* bufferBegi
                 Int32 sizeOfExpoent = snprintf(tempNumber + baseIndex, kTempCStringLength-baseIndex, "E%+d", (int)exponent);
                 baseIndex += sizeOfExpoent;
             }
-            TempStackCString numberPart((Utf8Char*)tempNumber, baseIndex);
-            GenerateFinalNumeric(formatOptions, bufferBegin, pSize, &numberPart, negative);
+            tempNumber[baseIndex] = 0;
```

Comment:
This code change seems to be identical to the previous change, now that we more than two lines of code doing same, consider creating a helper method, to be called in both places.